portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboardAuditGraph.testcase

```diff
@@ -0,0 +1,185 @@
+@component-name = "portal-content-dashboard"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Content Dashboard";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		JSONGroup.addGroup(groupName = "Test Site Name");
+
+		Navigator.openURL();
+
+		var categoryId1 = JSONCategory.addCategory(
+			groupName = "Global",
+			title = "Audience 1",
+			vocabularyName = "Audience");
+
+		var categoryId2 = JSONCategory.addCategory(
+			groupName = "Global",
+			title = "Stage 1",
+			vocabularyName = "Stage");
+	}
+
+	tearDown {
+		WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
```

Comment:
Line 29 until 37.
Take a look in other poshi tests, if it is inside of the else or not, please.